modules/core/src/main/java/org/apache/ignite/internal/processors/performancestatistics/FilePerformanceStatisticsReader.java

```diff
@@ -69,15 +69,15 @@
  * @see FilePerformanceStatisticsWriter
  */
 public class FilePerformanceStatisticsReader {
-    /** Default file read buffer size. */
-    private static final int DFLT_READ_BUFFER_SIZE = (int)(8 * U.MB);
-
     /** Uuid as string pattern. */
-    private static final String UUID_STR_PATTERN =
+    static final String UUID_STR_PATTERN =
         "[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}";
 
     /** File name pattern. */
-    private static final Pattern FILE_PATTERN = Pattern.compile("^node-(" + UUID_STR_PATTERN + ")(-\\d+)?.prf$");
+    static final Pattern FILE_PATTERN = Pattern.compile("^node-(" + UUID_STR_PATTERN + ")(-(\\d+))?.prf$");
+
+    /** Default file read buffer size. */
+    private static final int DFLT_READ_BUFFER_SIZE = (int)(8 * U.MB);
```

Comment:
Why this change? Please, revert moving this constant from 72 to 80 line.

Reply:
done